src/utils/user-role-utils.tsx

```diff
@@ -30,4 +45,13 @@ export default class UserRoleUtils {
 
     return UserRoleUtils.isAdmin() && adminPathname;
   };
+
+  /**
+   * Check if the logged-in user has devveloper role and is in admin route
```

Comment:
double v

Reply:
Also it would be good to reduce the duplication of code

Reply:
> Also it would be good to reduce the duplication of code

I fixed the typo, but the code is copied from current development to master, should be probably refactored in develop not here since its just a hotfix.